.ci_build_samples.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+cd samples
+for dir in */
+do
+    cd "$dir"
+    make
+    cd ..
+done
```

Comment:
Why are these commits part of this PR? Please re-base on latest master.

Also squash your 2 commits: no need for anybody to see that wchar was in "xlibc" first.
When squashing, keep the original commit title (which doesn't overflow, unlike the title of the last commit).

Reply:
Sorry about that. Been a while since I've had to do this and messed up. Should be better now.